ngraph/test/backend/roll.in.cpp

```diff
@@ -0,0 +1,149 @@
+//*****************************************************************************
+// Copyright 2017-2021 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "gtest/gtest.h"
+#include "ngraph/ngraph.hpp"
+#include "ngraph/opsets/opset7.hpp"
+#include "ngraph/runtime/tensor.hpp"
+#include "ngraph/shape.hpp"
+#include "runtime/backend.hpp"
+#include "util/all_close_f.hpp"
+#include "util/test_control.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+static string s_manifest = "${MANIFEST}";
+
+NGRAPH_TEST(${BACKEND_NAME}, roll_2d_input)
+{
+    Shape shape{4, 3};
+    auto x = make_shared<opset7::Parameter>(element::f32, shape);
+    auto shift = make_shared<opset7::Constant>(element::i64, Shape{1}, vector<int64_t>{1});
+    auto axes = make_shared<opset7::Constant>(element::i64, Shape{1}, vector<int64_t>{0});
+    auto f = make_shared<Function>(make_shared<opset7::Roll>(x, shift, axes), ParameterVector{x});
+
+    auto backend = runtime::Backend::create("${BACKEND_NAME}");
+
+    auto x_tensor = backend->create_tensor(element::f32, shape);
+    copy_data(x_tensor, vector<float>{1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12});
```

Comment:
Please, make these values really floating point. For example, "1.23, 2.35, 3.07, 4.0" etc